web/src/components/profileEdit/ProjectCardEdit.tsx

```diff
@@ -49,6 +49,33 @@ interface MinistryItem {
   code: string;
 }
 
+export const renderSubmitButton = (hasPendingEdit: any, isProvisioned: any, pristine: boolean) => {
```

Comment:
Good decision to extract this into a separate component. 

nit pick: I wonder if it could be extracted to its own file, maybe under utils, to avoid nested common logic within a single file.

Reply:
updated